test/integration/users_login_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+
+class UsersLoginTest < ActionDispatch::IntegrationTest
+  
+  def setup
+    @user = users(:michael)
+  end
+  test "login with invalid information" do
+    get login_path
+    assert_template 'sessions/new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.